src/main/java/sandbox/Sandbox.java

```diff
@@ -1,59 +1,62 @@
 package sandbox;
 
-import helpers.CurrencyExchanger;
-import helpers.DatePanel;
+import api.API;
+import com.toedter.calendar.JDateChooser;
 
 import javax.swing.*;
 import java.awt.*;
-import java.awt.event.*;
-import java.text.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Locale;
+import java.util.concurrent.TimeUnit;
 
-import static main.Main.HOME_CURRENCY;
 
-public class Sandbox extends JPanel
-{
+public class Sandbox extends JPanel {
+    private static final String HOME_CURRENCY = "USD";
     private final ArrayList<WhatIfPanel> whatIfs = new ArrayList<>();
     private final JScrollPane scrollPane;
-    private final JPanel whatIf;
-    private DatePanel specifiedDate;
+    private final JPanel whatIfPanel;
```

Comment:
whatIfPanel should be renamed since it is not a WhatIfPanel - maybe whatIfContainerPanel?